scripts/testimonials.js

```diff
@@ -3,15 +3,23 @@ var testimonialObjectArray = [
     name:             'Rick_Patci',
     quote:            'Lee is the best programmer ever. I am continually blown away by what he can do.',
     quoteAuthor:      'Rick Patci',
-    quoteAuthorLink:  'http://rickpatci.com'
+    quoteAuthorLink:  'https://github.com/patci'
```

Comment:
A lot of excess spacing in this file. Since your other script files are pretty tight, assuming you just haven't gotten to tidying up this one yet!